docs/conf.py

```diff
@@ -15,6 +15,7 @@
 import pkg_resources
```

Comment:
I guess this is no longer needed? lets remove it to unblock flake8 errors

Reply:
fixed